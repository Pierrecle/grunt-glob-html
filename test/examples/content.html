<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title></title>
    <!--script with while card-->
    <script src="scripts1/**/*.js"></script>
    <!--script with attr-->
    <script src="scripts2/*" some-attr="some-value"></script>
    <!--script without wildcard should be skipped-->
    <script src="scripts2/script4.js"></script>
    <script>
        console.log("inline script should not be touched");
    </script>
</head>
<body>
    <span>&quot;</span>
</body>
</html>