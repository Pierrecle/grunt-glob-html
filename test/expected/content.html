<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title></title>
    <!--script with while card-->
    <script src="scripts1/more/another-script.js"></script>
    <script src="scripts1/script1.js"></script>
    <script src="scripts1/script2.js"></script>
    <script src="scripts1/script3.js"></script>
    <!--script with attr-->
    <script src="scripts2/script4.js" some-attr="some-value"></script>
    <script src="scripts2/script5.js" some-attr="some-value"></script>
    <script src="scripts2/script6.js" some-attr="some-value"></script>
    <!--script without wildcard should be skipped-->
    <script src="scripts2/script4.js"></script>
    <script>
        console.log("inline script should not be touched");
    </script>
</head>
<body>
    <span>&quot;</span>
</body>
</html>